clang/lib/Sema/TreeTransform.h

```diff
@@ -13714,6 +13714,12 @@ TreeTransform<Derived>::TransformLambdaExpr(LambdaExpr *E) {
 
     // Capturing 'this' is trivial.
     if (C->capturesThis()) {
+      // We need ThisType here.
```

Comment:
This comment doesn’t really explain anything. I’d suggest explaining *why*, not *that*, we need a `ThisScope` here.